src/mail/model/MailUtils.ts

```diff
@@ -248,7 +248,12 @@ export function getMailboxName(logins: LoginController, mailboxDetails: MailboxD
 	if (!logins.isInternalUserLoggedIn()) {
 		return lang.get("mailbox_label")
 	} else if (isUserMailbox(mailboxDetails)) {
-		return logins.getUserController().props.defaultSender ?? logins.getUserController().userGroupInfo.mailAddress ?? lang.get("mailbox_label")
+		const userController = logins.getUserController()
+		return userController.userGroupInfo &&
```

Comment:
Yes, but getDefaultSender (and getDefaultSenderFromUser) assert mailGroupInfo.mailAddress as not null, but this case was allowing it to be null and returning something else, so that's why I didn't just use it. But we could?

Reply:
I suspect @ganthern did it out of abundance of caution. I think we should be consistent at least

Reply:
I think it should be fine, if we run into problems with that assertion we only need to change one place at least?